src/compound_targets.py

```diff
@@ -16,8 +15,9 @@
 
 @mod.capture(
     rule=(
-        "[{user.cursorless_range_connective}] <user.cursorless_primitive_target> | "
-        "<user.cursorless_primitive_target> {user.cursorless_range_connective} <user.cursorless_primitive_target>"
+        "<user.cursorless_primitive_target> | "
+        "[<user.cursorless_range_type>] {user.cursorless_range_connective} <user.cursorless_primitive_target> | "
+        "[<user.cursorless_range_type>] <user.cursorless_primitive_target> {user.cursorless_range_connective} <user.cursorless_primitive_target>"
```

Comment:
Haven't had a chance to DFA-test this one yet, but I worry this one could hit DFA because it adds an extra `user.cursorless_primitive_target` path

Reply:
This is no worse than what's already in develop. 
`[a] b` is no worse than `b | a b`

Reply:
K cool if DFA is fine for you I'm happy